src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -111,7 +111,7 @@ public CatalogObjectMetadataList create(
             @ApiParam(value = "Name of the object or empty when a ZIP archive is uploaded (All objects inside the archive are stored inside the catalog).") @RequestParam(required = false) Optional<String> name,
             @ApiParam(value = "Kind of the new object") @RequestParam String kind,
             @ApiParam(value = "Commit message") @RequestParam String commitMessage,
-            @ApiParam(value = "The content type of CatalogRawObject") @RequestParam String contentType,
+            @ApiParam(value = "The content type of CatalogRawObject") @RequestParam String objectContentType,
```

Comment:
Fix a bug during client generation in order to avoid duplication with an other contentType variable.